docassemble/AssemblyLine/data/questions/al_document.yml

```diff
@@ -11,13 +11,46 @@ features:
 generic object: ALDocumentBundle
 event: x.send_email_action_event
 code: |
-  email_success = x.send_email(to=action_argument('email'), editable=action_argument('editable'), template=action_argument('template'))
+  email_success = x.send_email(
+      to=action_argument('email'),
```

Comment:
This does technically expose the email address being sent to, to the user, it's just in the JS source; we could wrap this in a `value(...)`, and only have the email set in a variable in the interview, but that's still accessible to the user (I think redis is the only way to keep things really secret, and that feels like over-engineering).

It's probably fine, as long as that's clearly explained; like, that it doesn't show the email to the user, but doesn't keep it secret either.

Reply:
Yes, you're right. It is possible to hide variables, but I'm not sure if it would break some of our code (if we use any JS code that needs the interview value). But when describing this we'd need to explain that end user can get the email.

In theory an author could provide their own event and use a dummy or empty email address as well.